host-configs/docker/docker-linux-ubuntu16.04-x86_64-gcc@8.1.0.cmake

```diff
@@ -1,73 +1,100 @@
+# Compiler Spec: gcc@8.1.0
+#------------------------------------------------------------------------------
+if(DEFINED ENV{SPACK_CC})
+
+  set(CMAKE_C_COMPILER "/home/serac/serac_tpls/spack/lib/spack/env/gcc/gcc" CACHE PATH "")
+
+  set(CMAKE_CXX_COMPILER "/home/serac/serac_tpls/spack/lib/spack/env/gcc/g++" CACHE PATH "")
+
+  set(CMAKE_Fortran_COMPILER "/home/serac/serac_tpls/spack/lib/spack/env/gcc/gfortran" CACHE PATH "")
 
-set(CMAKE_CXX_COMPILER "/usr/bin/g++" CACHE PATH "")
+else()
 
-set(CMAKE_C_FLAGS "-pthread" CACHE PATH "")
+  set(CMAKE_C_COMPILER "/usr/bin/gcc" CACHE PATH "")
 
-set(CMAKE_CXX_FLAGS "-pthread" CACHE PATH "")
+  set(CMAKE_CXX_COMPILER "/usr/bin/g++" CACHE PATH "")
 
-set(ENABLE_CUDA OFF CACHE BOOL "")
+  set(CMAKE_Fortran_COMPILER "/usr/bin/gfortran" CACHE PATH "")
 
-#---------------------------------------
+endif()
+
+set(CMAKE_C_FLAGS "-pthread" CACHE STRING "")
+
+set(CMAKE_CXX_FLAGS "-pthread" CACHE STRING "")
+
+#------------------------------------------------------------------------------
 # MPI
-#---------------------------------------
-set(ENABLE_MPI "ON" CACHE PATH "")
+#------------------------------------------------------------------------------
 
 set(MPI_C_COMPILER "/usr/bin/mpicc" CACHE PATH "")
 
 set(MPI_CXX_COMPILER "/usr/bin/mpic++" CACHE PATH "")
 
-set(MPIEXEC_EXECUTABLE "/usr/bin/mpiexec" CACHE PATH "")
+set(MPI_Fortran_COMPILER "/usr/bin/mpif90" CACHE PATH "")
+
+set(MPIEXEC_EXECUTABLE "/usr/bin/mpirun" CACHE PATH "")
+
+set(MPIEXEC_NUMPROC_FLAG "-np" CACHE STRING "")
+
+set(ENABLE_MPI ON CACHE BOOL "")
+
+#------------------------------------------------------------------------------
+# Hardware
+#------------------------------------------------------------------------------
+
+#------------------------------------------------
+# Hardware Specifics
+#------------------------------------------------
+
+set(ENABLE_GTEST_DEATH_TESTS ON CACHE BOOL "")
+
+#------------------------------------------------------------------------------
+# TPLs
+#------------------------------------------------------------------------------
 
-#---------------------------------------
-# Library Dependencies
-#---------------------------------------
 set(TPL_ROOT "/home/serac/serac_tpls/gcc-8.1.0" CACHE PATH "")
 
 set(AXOM_DIR "${TPL_ROOT}/axom-0.4.0serac" CACHE PATH "")
 
 set(CONDUIT_DIR "${TPL_ROOT}/conduit-0.5.1p1" CACHE PATH "")
 
+set(MFEM_DIR "${TPL_ROOT}/mfem-4.2.0" CACHE PATH "")
+
 set(HDF5_DIR "${TPL_ROOT}/hdf5-1.8.21" CACHE PATH "")
 
 set(HYPRE_DIR "${TPL_ROOT}/hypre-2.18.2" CACHE PATH "")
 
 set(METIS_DIR "${TPL_ROOT}/metis-5.1.0" CACHE PATH "")
 
-set(NETCDF_DIR "${TPL_ROOT}/netcdf-c-4.7.4" CACHE PATH "")
-
 set(PARMETIS_DIR "${TPL_ROOT}/parmetis-4.0.3" CACHE PATH "")
 
+set(NETCDF_DIR "${TPL_ROOT}/netcdf-c-4.7.4" CACHE PATH "")
+
 set(SUPERLUDIST_DIR "${TPL_ROOT}/superlu-dist-6.1.1" CACHE PATH "")
 
-set(MFEM_DIR "${TPL_ROOT}/mfem-4.2.0" CACHE PATH "")
+# PETSC not build
```

Comment:
I assume this is an upstream spack thing, but `build` -> `built`

Reply:
Ack i thought i fixed that.  I'll change it then not rebuild the host-configs